ember-addon-main.js

```diff
@@ -17,25 +17,30 @@ module.exports = {
       );
     }
 
-    let pluginPath = require.resolve('./src/babel-plugin');
+    let staticBlocks = require.resolve(
+      '@babel/plugin-transform-class-static-block'
+    );
 
-    if (!hasPlugin(includer, pluginPath)) {
-      addPlugin(includer, pluginPath);
-    }
+    addPlugin(includer, staticBlocks);
 
-    // Used in ember-cli-htmlbars to get the location of templateCompiler without traversing this.addons (https://github.com/ember-cli/ember-cli-htmlbars/blob/6860beed9a357d5e948abd09754e8a978fed1320/lib/ember-addon-main.js#L264)
+    let pluginPath = require.resolve('babel-plugin-ember-template-compilation');
```

Comment:
I don't think any of this should be here. All ember apps already have this plugin. 

They get it via ember-cli-htmlbars or embroider. 

I think we should document that ember-template-imports has a minimum supported version of ember-cli-htmlbars and/or embroider and leave it at that.

Reply:
And we'd update ember-cli-htmlbars and embroider so there are releases that require the new-enough babel-plugin-ember-template-compilation.

Also, those updates would probably need to add the static-block babel plugin, in the same place where they currently add the other class feature plugins that are forced by legacy decorators.

Reply:
sounds good, I made "a" change here in this PR, which I expect to fail until we update ember-cli-htmlbars and embroider